src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2838,6 +2837,16 @@ public static boolean paxoTopologyRepairStrictEachQuorum()
         return conf.paxos_topology_repair_strict_each_quorum;
     }
 
+    public static Config.LegacyPaxosStrategy getLegacyPaxosStrategy()
+    {
+        return conf.legacy_paxos_strategy;
+    }
+
+    public static void setLegacyPaxosStrategy(Config.LegacyPaxosStrategy strategy)
```

Comment:
Do we need to be able to set this at runtime? (Noticed it's `volatile`...)

Reply:
It's not a hot prop so no we don't. I'll remove the volatile.

Reply:
Should I also remove the setter? It is unused.

Reply:
Might as well.